homeassistant/components/xiaomi_miio/fan.py

```diff
@@ -1133,9 +1052,8 @@ def __init__(self, name, device, entry, unique_id):
         super().__init__(name, device, entry, unique_id)
 
         self._device_features = FEATURE_FLAGS_AIRFRESH
+        self._supported_features = SUPPORT_SET_SPEED | SUPPORT_PRESET_MODE
```

Comment:
If this is a bug fix, we should do this in a separate PR so we can release it in a patch release.

The clean up of the legacy speed lists will go out in the next minor release as a breaking change.

Reply:
I think this the only line that should heed to be in the PR if the cleanup and unrelated change is reverted.

Reply:
See https://github.com/home-assistant/core/pull/54342